src/schema/submission.yaml

```diff
@@ -1233,9 +1265,7 @@ slots:
 
   library_preparation_kit_retail_name:
     description: >-
-      A unique identifier for the kit used to construct a genomic library.
-      This may include the vendor name, kit name and kit version
-      (e.g. Agilent sure select Human Exome V8, Twist RefSeq Exome, etc.)
+      A unique identifier for the kit used to construct a genomic library. This may include the vendor name, kit name and kit version (e.g. Agilent sure select Human Exome V8, Twist RefSeq Exome).
```

Comment:
```suggestion
      A unique identifier for the kit used to construct a genomic library. This may include the vendor name, kit name and kit version (e.g., Agilent sure select Human Exome V8, Twist RefSeq Exome).
```